frb_spectral_search/acf_func.py

```diff
@@ -150,20 +150,21 @@ def acf_bands(spectra, n, plot_range = None, band_size = 40, normalize = True, s
                 ax.set_xlim([0,plot_range])
                 if plot_range < 101:
                     ax.xaxis.set_minor_locator(MultipleLocator(1))
-        
+
         start += channel_sz
     ax.set_xlabel( "frequency lag (24 kHz channels)" )
     plt.legend(bbox_to_anchor=(1, 0), loc='lower right', ncol=1)
     plt.show()
     return all_acf, bands, test
+
 def plot_bands(all_acf, bands, remove_nan=True):
     if remove_nan:
         for i in range(len(all_acf)):
             b = np.array(all_acf[i])
             all_acf[i] = b[~np.isnan(b)]
     widths =[]
     for i in all_acf:
-    
+
         if i != []:
             widths.append(abs(hmhw_cauchy(i, exclude_zero =True))*.024) #converting channels to MHz
```

Comment:
Would rename to `widths_mhz` to be clear about units to ward off unit conversion problems!